docs/akkadian.rst

```diff
@@ -147,7 +147,7 @@ These two instance attributes are used for the ATFConverter.
 
    In[2]: from cltk.corpus.akkadian.file_importer import FileImport
 
-   In[3]: text_location = os.path.join('..', 'Akadian_test_texts', 'Akkadian.txt')
+   In[3]: text_location = os.path.join('..', 'Akkadian_test_texts', 'Akkadian.txt')
```

Comment:
I still see the same issue, that your users will not have direct access to `Akkadian_test_texts/Akkadian.txt`. Where would a user obtain a file like `Akkadian.txt`? from CDLI? Maybe give a url to let someone download it.

Reply:
Let me know if this most recent change has resolved this issue!